README.md

```diff
@@ -1,5 +1,14 @@
 # From zero to hero: end to end data applications with SQL and Jupyter
 
+## Set up
+
+First set up by following this guide: https://ploomber-contributing.readthedocs.io/en/latest/contributing/setup.html
+
+After cloning this repository and activating your environment, follow these steps:
+
+1. cd into your cloned repository and install the required packages from the requirements.txt file. To install, run `pip install -r requirements.txt` in your command line.
```

Comment:
I don't see a requirements.txt, this project is using a conda environment.yml

Reply:
Hi @jpjon 

You can use the conda env update command:

`conda env update --name <your env name> -f <your file>.yml`

or, if the environment you want to update is already activated, then

`conda env update -f <your file>.yml`

Reply:
Because the conda environment setup for this repository differs from the general contributing guidelines, I created additional documentation under `CONTRIBUTING.md`. This addition includes how to set up the environment for this repository. 

The `README.md` file has a reference link that points to the "Setup" section of the `CONTRIBUTING.md` file for initial setup. Please confirm if this is what you meant from our conversation @lfunderburk

Reply:
This is looking good! 

As a sanity check, can you try deactivating your current environment and following the steps to create and install the dependencies on a fresh environment?

The purpose of this is to test.

1. Run `conda deactivate`

`(sqlenv) path/to/repo$ conda deactivate`

This should change your environment to `base`

`(base)path/to/repo$`

From here, run

`conda env create -f environment.yml`

and

`conda activate sql-course`

then try to:

1. Build the docs locally `jupyterbook build course-material/`
2. Performing linting checks `pkgmt format` and `pkgmt lint`

Reply:
if any issues come up, it means the .yaml file is still missing key dependencies

Reply:
The current environment.yml file's "- duckdb" addition under dependencies results in a `ResolvePackageNotFound:
  - duckdb` when creating a new environment from the .yml file. 

Prior to syncing with the current main repo, every step ran fine except for `jupyter-book build`, which resulted in some errors but still allowed me to build.

Reply:
Fixed this. 

Can you merge main into this branch?